DICOMwebProxy/src/main/java/online/kheops/proxy/ohif/OHIFMetadataResource.java

```diff
@@ -92,18 +92,24 @@ private String ohifServerMetadata(AuthorizationToken authorizationToken) {
                 "      \"dicomWeb\": [\n" +
                 "         {\n" +
                 "            \"name\": \"DCM4CHEE\",\n" +
-                "            \"wadoUriRoot\": \"%s/wado\",\n" +
                 "            \"qidoRoot\": \"%s\",\n" +
                 "            \"wadoRoot\": \"%s\",\n" +
                 "            \"qidoSupportsIncludeField\": true,\n" +
                 "            \"imageRendering\": \"wadors\",\n" +
                 "            \"thumbnailRendering\": \"wadors\",\n" +
-                "            \"enableStudyLazyLoad\": true\n" +
+                "            \"enableStudyLazyLoad\": true,\n" +
```

Comment:
The rest of the parameters set the bulkdata requests up so that they are accepted by KHEOPS.